build.gradle

```diff
@@ -35,6 +36,10 @@ subprojects {
     repositories {
         mavenCentral()
     }
+
+    javafx {
+        modules 'javafx.graphics'
```

Comment:
All services depend on `Util`, which needs graphics, so I think this is okay.

Reply:
I don't see `util` having any dependencies in its [pom.xml](https://repo1.maven.org/maven2/com/gluonhq/attach/util/4.0.9/util-4.0.9.pom)

Reply:
UtilService has a dependency in javafx.graphics (it is deliberately [removed](https://github.com/gluonhq/attach/blob/master/gradle/mavenPublish.gradle#L133) from the pom). We assume Attach will be always used with JavaFX.
 
However, the poms of the Attach services don't show any transitive dependency between services (which causes the need of adding Util for instance to the list of dependencies). That's for another issue.

Reply:
> UtilService has a dependency in javafx.graphics (it is deliberately removed from the pom). We assume Attach will be always used with JavaFX.

It is removed from all the attach modules. In that case, it doesn't harm to have the plugin applied and a dependency added for all the modules.

> However, the poms of the Attach services don't show any transitive dependency between services (which causes the need of adding Util for instance to the list of dependencies)

That's because `util` is added as an `implementation` dependency instead of `api`. I can make the change in this PR if required.

Reply:
We can try it, not only for Util, but for all services that required by other services (i.e, Lifecycle).

Reply:
Do you want me to make that change in this PR, or is it better to take it up in a separate PR?

Reply:
Probably better in a separate PR